src/core/js/drawer.js

```diff
@@ -25,7 +25,12 @@ define(function(require) {
 	Adapt.once('app:dataReady', function() {
 		init();
 	})
-
+	Adapt.on("drawer:opened",function(){
+		var itemsLength = $(".drawer-holder").children().length;
+		if(itemsLength == 1) { 
+		      Adapt.trigger("resources:showResources"); 
```

Comment:
Should be ``Adapt.trigger(DrawCollection.models[0].eventCallback);`` or similar.